src/styles/contacts.styl

```diff
@@ -0,0 +1,35 @@
+@require 'settings/palette.styl'
+
+.contact
+    display flex
+    height 48px
```

Comment:
Can you use `rem` instead of `px` values please? It allows the whole UI to scale with the font size.